File: CS/MyWizardControl/MyNextButton/MyNextButton.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraWizard;
using DevExpress.XtraEditors.Drawing;
using DevExpress.Skins;

namespace WindowsApplication1
{
    public class MyNextButton : BackButton
    {
        public MyNextButton(WizardControl wizard)
            : base(wizard)
        {
        }

        protected override DevExpress.XtraEditors.ViewInfo.BaseStyleControlViewInfo CreateViewInfo()
        {
            return new MyBackButtonViewInfo(this);
        }

    }

}

File: VB/MyWizardControl/MyNextButton/MyNextButton.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraWizard
Imports DevExpress.XtraEditors.Drawing
Imports DevExpress.Skins

Namespace WindowsApplication1
	Public Class MyNextButton
		Inherits BackButton
		Public Sub New(ByVal wizard As WizardControl)
			MyBase.New(wizard)
		End Sub

		Protected Overrides Function CreateViewInfo() As DevExpress.XtraEditors.ViewInfo.BaseStyleControlViewInfo
			Return New MyBackButtonViewInfo(Me)
		End Function

	End Class

End Namespace
